src/models/deprecated.ts

```diff
@@ -0,0 +1,28 @@
+import { SearchConfig } from './core/SearchConfig';
+import { BaseSearchConfig } from './core/SearchConfig';
+import { SearchConfigWithApiKey } from './core/SearchConfig';
+import { SearchConfigWithToken } from './core/SearchConfig';
+
+/**
+ * @deprecated AnswersConfig is deprecated and has been replaced by SearchConfig
```

Comment:
Could you try using an `{@link SearchConfig}` so the documentation pages have a link? also below